seed/challenges/01-front-end-development-certification/advanced-bonfires.json

```diff
@@ -324,9 +324,15 @@
       "id": "a2f1d72d9b908d0bd72bb9f6",
       "title": "Make a Person",
       "description": [
-        "Fill in the object constructor with the methods specified in the tests.",
-        "Those methods are getFirstName(), getLastName(), getFullName(), setFirstName(first), setLastName(last), and setFullName(firstAndLast).",
-        "All functions that take an argument have an arity of 1, and the argument will be a string.",
+        "Fill in the object constructor with the methods specified below:",
+        "getFirstName()",
+        "getLastName()",
+        "getFullName()",
+        "setFirstName(first)",
+        "setLastName(last)",
+        "setFullName(firstAndLast)",
+        "Run the tests to see the expected output for each method.",
+        "For the functions that take an argument, they should take only one string as an argument.",
```

Comment:
`The functions that take an argument, must accept only one and it has to be a string.`